django_perf_rec/sql.py

```diff
@@ -30,6 +30,11 @@ def sql_fingerprint(query):
 
 
 def sql_recursively_simplify(node):
+
+    # Erase volatile part of PG cursor name
+    if node.tokens[0].value.startswith('"_django_curs_'):
+        node.tokens[0].value = '"_django_curs_#"'
```

Comment:
I think this would make more sense underneath the savepoints block, as then the code is structured from 'common to rare' usage. Also the blank line above it is not needed.